docs/hw/cad-concept-schematics.md

```diff
@@ -0,0 +1,16 @@
+---
+layout: default
+title: CAD concept schematics
+parent: Operating System
+nav_order: 230
+---
+## Concept Schematics
+
+### BitBox Base Concept Render
+![BitBox Base 1](base_render_angle_1.png "Concept Render of BitBox Base")
+___
+### BitBox Base Concept Render Top View
+![BitBox Base 2](base_render_top_dimensions_1.PNG "Concept Render of BitBox Base - top view")
```

Comment:
I would also prefer to have the file extensions in lower-case here and for the next image.

Reply:
agreed. fixed.